From 763ccca169e7215e3247bb502d90342e86a87337 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Wed, 8 Mar 2023 15:33:59 +0100
Subject: [PATCH] arm64: defconfig: add typec support in addons fragment

Add Type-C UCSI config flags in addons fragment.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I819c496a795fdb14b952c99a6afb3759dbea439b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/369507
Reviewed-by: Pankaj DEV <pankaj.dev@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Pankaj DEV <pankaj.dev@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 59c5a849dbf1..01786fc79508 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -94,7 +94,9 @@ CONFIG_STMMAC_PLATFORM=y
 CONFIG_TEE_REMOTEPROC=y
 CONFIG_TOUCHSCREEN_GOODIX=y
 CONFIG_TOUCHSCREEN_ILI210X=y
+CONFIG_TYPEC_UCSI=m
 CONFIG_UBIFS_FS=y
+CONFIG_UCSI_STM32G0=m
 CONFIG_USB_DWC3_STM32=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ONBOARD_HUB=y
-- 
2.39.5

